README.md

```diff
@@ -0,0 +1,30 @@
+# arweave-checker
+Arweave Checker Module
+
+## Development
+
+Install [Zinnia CLI](https://github.com/filecoin-station/zinnia).
+
+```bash
+$ # Lint
+$ npx lint
+$ # Run module
+$ zinnia run main.js
+$ # Test module
+$ zinnia run test.js
+```
+
+## Release
+
+On a clean working tree, run the following command:
+
+```bash
+$ ./release.sh <semver>
+$ # Example
+$ ./release.sh 1.0.0
+```
+
+Use GitHub's changelog feature to fill out the release notes.
+
+Publish the new release and let the CI/CD workflow upload the sources
+to IPFS & IPNS.
```

Comment:
These instructions don't seem correct to me - there is no `release.sh` script. Let's remove them from this PR and then document the release process after we figure out all details about it.